services/github-service/src/main/java/org/kontinuity/catapult/service/github/api/NoSuchWebhookEventException.java

```diff
@@ -0,0 +1,16 @@
+package org.kontinuity.catapult.service.github.api;
+
+/**
+ * Indicates a specified Webhook event does not exist.
+ * 
+ * @author <a href="mailto:rmartine@redhat.com">Ricardo Martinelli de Oliveira</a>
+ */
+public class NoSuchWebhookEventException extends RuntimeException {
```

Comment:
It should be used to return to the user that he sent an invalid event, but with the change of GitHubWebhookEvent to an enum this Exception is useless now. I'm removing it.